pkgs/by-name/re/redwax-tool/package.nix

```diff
@@ -1,19 +1,19 @@
-{ lib, stdenv, fetchsvn
+{ lib, stdenv, fetchurl
 , autoreconfHook, pkg-config, txt2man, which
-, openssl, apr, aprutil
+, openssl, nss, p11-kit, libical, ldns, unbound, apr, aprutil
 }:
 
 stdenv.mkDerivation rec {
   pname = "redwax-tool";
-  version = "0.9.1";
+  version = "0.9.9";
 
-  src = fetchsvn {
-    url = "https://source.redwax.eu/svn/redwax/rt/redwax-tool/tags/redwax-tool-${version}/";
-    sha256 = "sha256-MWSB1AkkRS18UUHGq8EWv3OIXPSVHpmrdD5Eq1VdbkA=";
+  src = fetchurl {
+    url = "https://archive.redwax.eu/dist/rt/redwax-tool-${version}/redwax-tool-${version}.tar.gz";
+    hash = "sha256-gA9YWR3v66lSn4KkwG49l7V4/HUTrBXXl1uDCLr1tu8=";
   };
 
   nativeBuildInputs = [ autoreconfHook pkg-config txt2man which ];
-  buildInputs = [ openssl apr aprutil ];
+  buildInputs = [ openssl nss p11-kit libical ldns unbound apr aprutil ];
```

Comment:
The configure script does not find nss, p11-kit.
```
checking for NSS_Initialize... no
checking for p11_kit_modules_load_and_initialize... no
```
Most likely you need to adjust the `configureFlags`.

Reply:
The configure.ac uses pkgconfig exclusively for all libraries, so there definitely shouldn't be any fiddling with CFLAGS.

Can you confirm what you're seeing further up for the pkg-config library detection, and what does config.log say?

Reply:
The full build log is here https://paste.fliegendewurst.eu/lHs4jb - I'll see if I can get config.log as well.

Reply:
config.log https://paste.fliegendewurst.eu/Y4SFna

Reply:
yes. `configureFlags = [ "--with-openssl" "--with-nss" ... ];`, at the same level as `src` and `buildInputs`.

Reply:
Something like the commit below?

Reply:
That's awesome, much appreciated thank you.